packages/core/parcel-bundler/src/assets/CoffeeScriptAsset.js

```diff
@@ -9,7 +10,10 @@ class CoffeeScriptAsset extends Asset {
 
   async generate() {
     // require coffeescript, installed locally in the app
-    let coffee = await localRequire('coffeescript', this.name);
+    let coffee = await localRequire(
+      'coffeescript',
+      path.join(this.options.rootDir, 'index')
```

Comment:
Kind of wanna get rid of this `path.join` but not sure how to approach it.
I've seen it at various other places in the codebase as well.